src/config.js

```diff
@@ -42,18 +42,11 @@ export const flags = {
     default: true,
     icon: '🐳',
   },
-  modularpipeline: {
```

Comment:
Removing the `modularpipeline` flag should be another PR I think as it seems unrelated? If so revert this here I think.

Reply:
agree with @bru5 , not sure why the modular pipeline flag is removed here? Is this something done unintentionally from a merge conflict? 

Also speaking of, I think a feature flag needs to be set up for this feature?